### PR TITLE
Changing logo for kuonsamwiki

### PR DESCRIPTION
**Pull request by samkuonssp**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	die( 'Not an entry point.' );
 }
 
-$wmgUploadHostname = "static.miraheze.org";
+$wmgUploadHostname = "//$wmgUploadHostname/kuonsamwiki/b/bc/My_Handsome.png";
```
Comment (Reception123) [inline LocalSettings.php]: There is no reason to change this, please leave this as static.miraheze.org.

File: LocalSettings.php [hunk 2]
```diff
@@ -5920,6 +5920,7 @@
 		'wisdomwikiwiki' => "//$wmgUploadHostname/wisdomwikiwiki/0/02/WWlogo.png",
 		'wisdomsandboxwiki' => "//$wmgUploadHostname/wisdomsandboxwiki/b/be/Sandbox_Logo.png",
 		'wishwiki' => "//$wmgUploadHostname/wishwiki/c/c6/Tamers_internet.png",
+		'kuonsamwiki' => "//$wmgUploadHostname/kuonsamwiki/b/bc/My_Handsome.png",
```
Comment (Reception123) [inline LocalSettings.php]: this should be in alphabetical order

Comment (samkuonssp) [reply LocalSettings.php]: Oh ok, should I change and request to commit again?

Comment (Reception123) [reply LocalSettings.php]: You can just edit the current commit.

Comment (Reception123) [reply LocalSettings.php]: @samkuonssp If you're unsure you could just make a request on https://phabricator.miraheze.org

File: LocalSettings.php [hunk 3]
```diff
@@ -5858,6 +5858,7 @@
 		'sterbalfamilyrecipeswiki' => "//$wmgUploadHostname/sterbalfamilyrecipeswiki/c/c4/Logo-135x135.jpg",
 		'stormfmwiki' => "//$wmgUploadHostname/stormfmwiki/1/18/Stormlogo_small.png",
 		'studynotekrwiki' => "//$wmgUploadHostname/studynotekrwiki/b/b3/Imageedit_6_7597747851.gif",
+		'kuonsamwiki' => "//$wmgUploadHostname/kuonsamwiki/b/bc/My_Handsome.png",
```
Comment (Reception123) [inline LocalSettings.php]: Still not alphabetical order :( It should be under current line 5761, so _under_ kstartupswiki

**Pull request merged**